src/app/devices/sol/poweralert/poweralert.component.html

```diff
@@ -0,0 +1,8 @@
+<h2 mat-dialog-title>Device is disconnected</h2>
+<mat-dialog-content class="mat-typography">
+    Are you sure you to connect the device ?
```

Comment:
remove space after device

Reply:
This doesn't read correctly. Is it supposed to say "Are you sure you want to connect to the device?"

Reply:
done